crates/sargon-uniffi/src/core/result/result.rs

```diff
@@ -33,6 +33,11 @@ pub trait FromInternalResult<InternalType, Type> {
     fn into_result(self) -> Result<Type>;
 }
 
+/// Utility trait to map `InternalResult` to `Result`
+pub trait FromInternalIterResult<InternalType, Type> {
```

Comment:
Uh this is not so great, but not too bad. Instead of `into_result()`  on `Result<Vec<T>>` we need to call `into_iter_result()`....